Pipfile

```diff
@@ -37,6 +38,8 @@ coverage = ">=4.5.0"
 codecov = ">=2.0.0"
 faker = ">=0.8.14"
 pylint-django = "==2.0.9"
+pydocstyle = "~=3.0.0"
```

Comment:
Open issue for this: https://gitlab.com/pycqa/flake8-docstrings/issues/36
I'll revert this change if the issue is resolved before this PR gets merged.